velox/functions/sparksql/tests/ComparisonsTest.cpp

```diff
@@ -112,6 +125,88 @@ TEST_F(ComparisonsTest, equalto) {
   EXPECT_EQ(equalto<double>(kInf, kNaN), false);
 }
 
+TEST_F(ComparisonsTest, equalToAndEqualNullSafeForArray) {
```

Comment:
I wonder if it can improve the readability by separating the tests for equalto and equalnullsafe. Since below tests exist, shall we add complex-type tests into them?

> TEST_F(ComparisonsTest, equaltonullsafe)
TEST_F(ComparisonsTest, equalto)

Reply:
Sure, thanks.